src/main/java/seedu/address/logic/commands/FindCommand.java

```diff
@@ -26,11 +23,15 @@ public FindCommand(NameContainsKeywordsPredicate predicate) {
     }
 
     @Override
-    public CommandResult execute(Model model) {
+    public CommandResult execute(ItemModel model) {
+        /* Original Code
         requireNonNull(model);
         model.updateFilteredPersonList(predicate);
         return new CommandResult(
                 String.format(Messages.MESSAGE_PERSONS_LISTED_OVERVIEW, model.getFilteredPersonList().size()));
+
+         */
+        return new CommandResult("dummy");
```

Comment:
The find logic is actually already implemented in item list, but I can implement the find command as well :)

Reply:
Sure, that would be great :)